conf/modules.config

```diff
@@ -155,6 +155,19 @@ process {
         ]
     }
 
+    withName: PRINSEQPLUSPLUS {
+        ext.args =  [
+                params.shortread_complexityfilter_prinseqplusplus_mode == 'dust' ? "-lc_dust=${params.shortread_complexityfilter_prinseqplusplus_dustscore}" : "-lc_entropy=${params.shortread_complexityfilter_entropy}",
+                "-trim_qual_left=0 -trim_qual_left=0 -trim_qual_window=0 -trim_qual_step=0"
```

Comment:
I don't know the program so I can't really judge these options. I'm guessing you're configuring dust and disabling quality-based trimming.

Reply:
Me neither.


When I look at the help message those are the only values with defaults, but we don't want to do so any read trimming here, so I'm deactivating by setting to 0 for safety

So here I'm saying if user selects dust, use the dust argument, and if not the entropy argument. In both cases we append to it the disabling of trimming